packages/shared/lib/app.ts

```diff
@@ -30,16 +31,17 @@ export const mnemonic = writable<Array<string>>(null)
 
 interface SendParams {
     amount: number
+    unit?: Unit
     address: string
     message: string
     isInternal: boolean
 }
 
 /**
- * Input paramaters for sending transactions
+ * Input parameters for sending transactions
  */
-export const sendParams = writable<SendParams>({ amount: 0, address: '', message: '', isInternal: false })
-export const clearSendParams = (isInternal = false) => sendParams.set({ amount: 0, address: '', message: '', isInternal })
+export const sendParams = writable<SendParams>({ amount: 0, unit: undefined, address: '', message: '', isInternal: false })
```

Comment:
Is it best to leave this as `undefined` or should we use `Unit.i` or `Unit.Mi` here?